### PR TITLE
[BEAM-14370] [Website] Add new page about appache beam

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-14370]
- add new menu top-level navigation item
- add about in both content and layouts
- add new image

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @nausharipov

File: website/www/site/content/en/about/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+- <u>Unified Batch and Streaming Programming Model</u>
+
+Apache Beam provides flexibility to express the business logic just once and execute it for both batch and streaming data pipelines, on-premises via OSS runners or in the cloud via managed services such as Google Cloud Dataflow or AWS Kinesis Data Analytics.  Apache Beam unifies multiple data processing engines and SDKs around its distinctive Beam model. This offers a way to easily create a large-scale common data infrastructure across different applications that consume the data.<br /><br />
```
Comment (nausharipov) [inline website/www/site/content/en/about/_index.md:37]: seems like there's an extra space here: "... Analytics.  Apache Beam unifies ..." 

You can fix it by formatting the code.

File: website/www/site/layouts/about/baseof.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<!DOCTYPE html>
+<html lang="{{ .Site.Language.Lang }}" class="no-js">
+<head>
+  {{ partial "head.html" . }}
+</head>
+<body class="body" data-spy="scroll" data-target=".page-nav" data-offset="0">
+{{ partial "header.html" . }}
+<div class="clearfix container-main-content">
+  <nav class="page-nav clearfix" data-offset-top="90" data-offset-bottom="500">
```
[the remark this answers is not in the document]
Comment (bullet03) [reply website/www/site/layouts/about/baseof.html:21]: done

File: website/www/site/content/en/about/_index.md (hunk 1 above)
Comment (bullet03) [reply website/www/site/content/en/about/_index.md]: done

File: website/www/site/content/en/about/_index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+-->
+
+# About
+
+## Why Apache Beam?
+
+Apache Beam is an open-source, unified programming model for batch and streaming data processing pipelines that simplifies large-scale data processing dynamics. Thousands of organizations around the world choose Apache Beam due to its unique data processing features, proven scale, and powerful yet extensible capabilities.
+
+<div style="display: flex; justify-content: center">
+  <img src="/images/batch_stream_processing.png" alt="Batch and Stream Processing">
```
[the remark this answers is not in the document]
Comment (bullet03) [reply website/www/site/content/en/about/_index.md:26]: optimized and made in smaller

**Comment by dariabezkorovaina** [comment]
@pcoet how does this PR look to you?:)

**Review by pcoet — APPROVED** [review]

**Pull request merged**